.github/workflows/release.yml

```diff
@@ -36,11 +36,17 @@ jobs:
 
       - name: Publish to VSCode Marketplace
         if: steps.changesets.outputs.published == 'true'
+        working-directory: ./packages/vscode
         run: |
-          npm i -g vsce
-          cd packages/vscode
-          npm install
-          vsce publish -p ${{ secrets.VSCE_TOKEN }}
+          npm i
+          npx vsce publish -p ${{ secrets.VSCE_TOKEN }}
+
+      - name: Publish to OpenVSX
+        if: steps.changesets.outputs.published == 'true'
+        working-directory: ./packages/vscode
+        run: |
+          npm i
```

Comment:
```suggestion
```

Reply:
I actually tested locally and without running `npm i` packaging does not work. Did it work for you?